combirepo/temporaries.py

```diff
@@ -165,7 +169,7 @@ def mount_firmware(firmware_path):
         image = images[0]
         mount_image(root, image)
     # For 3-parts images:
-    elif len(images) == 3:
+    elif len(images) >= 3:
         __mount_images_triplet(images, root)
     else:
         raise Exception("This script is able to handle only all-in-one or "
```

Comment:
Seems that we need to adjust the Exception message here.

Reply:
sure